microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/utils/AnnotationMemberInfo.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+*
+* This program and the accompanying materials are made available under the
+* terms of the Eclipse Public License v. 2.0 which is available at
+* http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+* which is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4mp.jdt.core.utils;
+
+import org.eclipse.lsp4j.Range;
+
+/**
+ * Utility class to store the annotation member info such as the value and the
+ * range
+ *
+ * @author AlexXuChen
+ *
+ */
+public class AnnotationMemberInfo {
+	private String annotationMemberValue;
+	private Range propertyKeyRange;
+
+	public AnnotationMemberInfo(String annotationMemberValue, Range propertyKeyRange) {
+		this.annotationMemberValue = annotationMemberValue;
+		this.propertyKeyRange = propertyKeyRange;
+	}
+
+	public String getAnnotationMemberValue() {
```

Comment:
rename to `getValue()` and add a Javadoc.

Reply:
This will conflict with the object getValue

Reply:
Sorry I don't understand.

Reply:
Object has a getValue, so i believe getMemberValue will avoid conflict